index.html

```diff
@@ -0,0 +1,464 @@
+                <p>Six years of experience in playing Chess, nationally and internationally</p>
+              </div>
+            </div>
+            </div>
+            <div class="col-12 col-md-6">
+              <div class="row mb-5 p-4 bg-white">
+              <div class="col-3 speaker-block pr-0 pl-0">
+                <img class="img-fluid rounded speaker-avatar" src="./assets/images/KevinOne.jpeg" alt="Speaker">
+              </div>
+              <div class="col-9 speaker-block pr-0 pl-4">
+                <h3 class="mb-1 section-title font-weight-bold">Kevin Koome</h3>
+                <p class="red font-italic">World Chess Champion</p>
+                <hr class="hr-speakers ml-0 mb-3">
+                <p>Six years of experience in playing Chess, nationally and internationally</p>
+              </div>
+            </div>
+            </div>
+            <div class="col-12 col-md-6">
+              <div class="row mb-5 p-4 bg-white">
+              <div class="col-3 speaker-block pr-0 pl-0">
+                <img class="img-fluid rounded speaker-avatar" src="./assets/images/KevinTwo.jpeg" alt="Speaker">
+              </div>
+              <div class="col-9 speaker-block pr-0 pl-4">
+                <h3 class="mb-1 section-title font-weight-bold">Kevin Koome</h3>
+                <p class="red font-italic">World Chess Champion</p>
+                <hr class="hr-speakers ml-0 mb-3">
+                <p>Six years of experience in playing Chess, nationally and internationally</p>
+              </div>
+            </div>
+            </div>
+          </div>
+        </section>
+
+
+      <!--partner section-->
+      <section class="container-fluid partners-section text-white bg-dark-custom">
+        <div class="row pt-3 mb-3">
+          <div class="col-12">
+            <h3 class="text-center section-title">Partner</h3>
+            <hr class="hr-title">
+          </div>
+        </div>
+        <div class="partners">
+          <figure>
+            <img src="./assets/images/chessclub.png" alt="Partner">
+          </figure>
+          <figure>
+            <img src="./assets/images/europeanchess.png" alt="Partner">
+          </figure>
+          <figure>
+            <img src="./assets/images/fide.png" alt="Partner">
+          </figure>
+          <figure>
+            <img src="./assets/images/professionalchess.png" alt="Partner">
+          </figure>
+          <figure>
+            <img src="./assets/images/uschess.png" alt="Partner">
+          </figure>
+        </div>
+      </section>
+    </main>
+
+
+    <!--footer-->
+    <footer class="container-fluid p-4 bg-light">
+      <div class="row">
+        <div class="col-6">
+          <figure>
+            <img class="img-fluid" src="./assets/images/creative.jpg" alt="Creative Common">
+          </figure>
+        </div>
+        <div class="col-6">
+          <p>2021 Creative Commons Korea <br>
+          Some Rights Reserved </p>
```

Comment:
### Original design
![original-footer](https://user-images.githubusercontent.com/80149314/142494386-da1a56e9-bf78-4283-8d89-fa6e7893eba8.jpg)

### Your design
![review-footer](https://user-images.githubusercontent.com/80149314/142494449-8dd45d6d-9e88-4396-b7b2-581209f39184.jpg)

The size of the Creative Commons differ a lot and there's so much space between the logo and the text in your case and the elements are not centered like in the given design. Please make the requested changes

Reply:
I have these changes too thank you for highlighting :+1: